compiler/ballerina-parser/src/main/java/io/ballerina/compiler/internal/parser/BallerinaParserErrorHandler.java

```diff
@@ -1209,7 +1212,7 @@ protected Result seekMatch(ParserRuleContext currentCtx, int lookahead, int curr
                 lookahead++;
                 isEntryPoint = false;
             }
-
+          
```

Comment:
```suggestion

```

Reply:
Here we need the newline after the `}`. It is good to have a newline after an if-else block etc.
What I suggested was to remove the extra whitespaces at the beginning of the newline. :)